onnxruntime/core/providers/openvino/ov_versions/data_ops.h

```diff
@@ -0,0 +1,69 @@
+#pragma once
+#include <unordered_set>
+
+namespace onnxruntime {
+namespace openvino_ep {
+
+typedef enum version_num { 
```

Comment:
Let us make sure wherever there's a **class, enum and a struct**, we should have the naming taken care of. I mean where to capitalize the names.

Ex: typedef enum **Version_Num** instead of typedef enum version_num

Reply:
I think VersionNum will be better, No need of underscores

Reply:
Resolved